crypto/init.c

```diff
@@ -498,6 +484,9 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
         aloaddone = 1;
     }
 
+    if (!ossl_init_casecmp())
```

Comment:
Kind of wondering about the need for this.  Why not let it lazily init itself?  This will simplify things a little.

Reply:
I prefer explicit initialization to lazy one. I want to keep it.